dev/dependencyList

```diff
@@ -182,7 +182,7 @@ swagger-jaxrs2/2.2.1//swagger-jaxrs2-2.2.1.jar
 swagger-models/2.2.1//swagger-models-2.2.1.jar
 trino-client/411//trino-client-411.jar
 units/1.7//units-1.7.jar
-vertx-core/4.5.1//vertx-core-4.5.1.jar
+vertx-core/4.5.3//vertx-core-4.5.3.jar
```

Comment:
Only updating the `vertx-core` version may not guarantee compatibility with `vertx-grpc`.
The vertx dependency is introduced by jetcd. How about directly upgrading the jetcd version?

Reply:
HI @cxzl25 , I tried upgrading jetcd to 0.8+, but it gave the error and asked to have java 11+
I agree to you we can try upgrading vertx-grpc itself